modules/ROOT/pages/rest-sdk/simple-composition.adoc

```diff
@@ -0,0 +1,80 @@
+= Simple Composition
+ifndef::env-site,env-github[]
+include::../_attributes.adoc[]
+endif::[]
+
+Simple composition can be used to build a DSL through declarative metadata.
```

Comment:
I don't think our users know what the DSL is. I suggest speaking about the use-cases here e.g. input/output transformations, hiding of parameters, etc. Here are all use-cases - https://docs.google.com/presentation/d/1MW1S8UPGULOxAXqO8R3wfoPrN1JvrQYLRC8mUUcp52E/edit#slide=id.gc7da8b0ac0_0_0